### DISCUSSION TITLE
Is authorisation a typo?

### DISCUSSION THREAD

**Discussion by myl7** [spencerwooo/onedrive-vercel-index#363] [General]
When translating I found many `authorisation`. `authorization` should be a much more common spelling. Though `authorisation` does exist, it feels strange and is rarely used.

**Comment by spencerwooo** [comment]
lol, I would feel offended if I were British.

**Reply by spencerwooo** [reply]
I just came back from Scotland, it's hard to change these spelling habits when everyone around you uses `s` instead of `z`.
